Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["delta-inspect", "rust", "python"]
+members = ["delta-inspect", "rust", "python", "deltalake/*"]
```

Comment:
Interesting syntax :thinking:

I was actually refactoring the `deltalake` meta crate and `deltalake-core` crates over the weekend end got the impression that such wizardry was not possible to nest crates like this :thinking: